src/aap_eda/wsapi/consumers.py

```diff
@@ -179,6 +189,21 @@ async def handle_actions(self, message: ActionMessage):
         logger.info(f"Start to handle actions: {message}")
         await self.insert_audit_rule_data(message)
 
+    async def _set_log_tracking_id(self, data: dict):
+        activation_instance_id = data.get("activation_id")
+        if activation_instance_id:
+            activation = await self.get_activation(activation_instance_id)
+            assign_log_tracking_id(activation.log_tracking_id)
+
+    async def _set_request_id(self):
+        headers = {
+            k.decode().lower(): v.decode()
```

Comment:
We don't need to decode all keys and values.

Reply:
we only need headers to get `x-request-id`